<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
 <HEAD>
   <TITLE> [Sonata-users] deleted profiles
   </TITLE>
   <LINK REL="Index" HREF="http://lists.berlios.de/pipermail/sonata-users/2008-December/index.html" >
   <LINK REL="made" HREF="mailto:sonata-users%40lists.berlios.de?Subject=Re%3A%20%5BSonata-users%5D%20deleted%20profiles&In-Reply-To=%3C494ACB7B.5030209%40gmail.com%3E">
   <META NAME="robots" CONTENT="index,nofollow">
   <style type="text/css">
       pre {
           white-space: pre-wrap;       /* css-2.1, curent FF, Opera, Safari */
           }
   </style>
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="000864.html">
   <LINK REL="Next"  HREF="000866.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[Sonata-users] deleted profiles</H1>
    <B>Sebastian</B> 
    <A HREF="mailto:sonata-users%40lists.berlios.de?Subject=Re%3A%20%5BSonata-users%5D%20deleted%20profiles&In-Reply-To=%3C494ACB7B.5030209%40gmail.com%3E"
       TITLE="[Sonata-users] deleted profiles">basti.kr at gmail.com
       </A><BR>
    <I>Thu Dec 18 23:15:23 CET 2008</I>
    <P><UL>
        <LI>Previous message: <A HREF="000864.html">[Sonata-users] New one here, plugins framework :D
</A></li>
        <LI>Next message: <A HREF="000866.html">[Sonata-users] deleted profiles
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#865">[ date ]</a>
              <a href="thread.html#865">[ thread ]</a>
              <a href="subject.html#865">[ subject ]</a>
              <a href="author.html#865">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>Hi all,
I accidentally removed all profiles and when I start sonata now it
periodically outputs &quot;list index out of range&quot;. When I try to open the
preferences window, the following error is raised:

Traceback (most recent call last):
  File &quot;/usr/lib/python2.6/site-packages/sonata/main.py&quot;, line 5722, in
on_prefs
    self.prefs_profile_chosen(profiles, nameentry, hostentry, portentry,
passwordentry, direntry)
  File &quot;/usr/lib/python2.6/site-packages/sonata/main.py&quot;, line 6255, in
prefs_profile_chosen
    nameentry.set_text(str(self.profile_names[prefs_profile_num]))
IndexError: list index out of range

Is there any possibility to restore the default profile?

Sebastian

</PRE>

<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="000864.html">[Sonata-users] New one here, plugins framework :D
</A></li>
	<LI>Next message: <A HREF="000866.html">[Sonata-users] deleted profiles
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#865">[ date ]</a>
              <a href="thread.html#865">[ thread ]</a>
              <a href="subject.html#865">[ subject ]</a>
              <a href="author.html#865">[ author ]</a>
         </LI>
       </UL>

<hr>
<a href="https://lists.berlios.de/mailman/listinfo/sonata-users">More information about the Sonata-users
mailing list</a><br>
</body></html>
